lib/Ed25519KeyPair.js

```diff
@@ -78,6 +78,67 @@ class Ed25519KeyPair extends LDKeyPair {
    * @returns {Promise<Ed25519KeyPair>} Generates a key pair.
    */
   static async generate(options = {}) {
+    // FIXME: determine exact release that included ed25519 key generation
```

Comment:
Does this need to be addressed?

Reply:
I found this commit which tells me that it landed in 12.0.0, so I'm removing the fixme: https://github.com/nodejs/node/commit/3a9592496c

From: https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V12.md#semver-minor-commits